### ISSUE TITLE
unit tests give error "'ElasticTensor' object has no attribute 'kg_average'"

### ISSUE THREAD

**Issue by computron** [hackingmaterials/atomate#92]
## System

* Version: master

## Summary

* Running the unit tests locally gives an error in (atomate.vasp.workflows.tests.test_elastic_workflow.TestElasticWorkflow)
* Issue 1: the test fails
* Issue 2: the test does *not* fail on CircleCI - why? Is it due to older pymatgen on CircleCI?

## Example code

Run the unit tests

## Error message

```
Traceback (most recent call last):
  File "/Users/ajain/Documents/code_matgen/atomate/atomate/vasp/workflows/tests/test_elastic_workflow.py", line 155, in test_wf
    self._check_run(d, mode="elastic analysis")
  File "/Users/ajain/Documents/code_matgen/atomate/atomate/vasp/workflows/tests/test_elastic_workflow.py", line 126, in _check_run
    c_ij = np.array(d['elastic_tensor'])
TypeError: 'NoneType' object has no attribute '__getitem__'
```

Also
```
Traceback (most recent call last):
  File "/Users/ajain/Documents/code_matgen/fireworks/fireworks/core/rocket.py", line 229, in run
    m_action = t.run_task(my_spec)
  File "/Users/ajain/Documents/code_matgen/atomate/atomate/vasp/firetasks/parse_outputs.py", line 242, in run_task
    kg_average = result.kg_average
AttributeError: 'ElasticTensor' object has no attribute 'kg_average'
```

## Suggested solution (if any)

* In March 15 @montoyjh seemed to remove the kg_average parameter from the ElasticTensor object. This should be updated...

**Comment by computron** [comment]
Note to @montoyjh : I am pretty sure CircleCI succeeds because the atomate reqs are still PMG 4.7.2. After fixing can you update the requirements (both setup.py and requirements.txt) to PMG 4.7.3?

**Comment by montoyjh** [comment]
Yep, should have it done later today.

**Comment by montoyjh** [comment]
Resolved via 4a07dd923abd63c6d3733ad413d0119b9a5a0018.

**Comment by montoyjh** [comment]
Resolved via 4a07dd923abd63c6d3733ad413d0119b9a5a0018.
